src/pages/fileCentricCart/cartController.js

```diff
@@ -32,7 +34,7 @@ const cartController = () => {
       isLoading={false}
       fileIDs={ids}
       updateSortOrder={updateSortOrder}
-      defaultSortCoulmn={table.defaultSortField || ''}
+      defaultSortCoulmn={defaultSortColumnValue}
```

Comment:
you can directly write the conditional logic in here, instead of creating a new variable